index.html

```diff
@@ -146,48 +185,55 @@ <h1 class="about-me-title">About me</h1>
-      <li class="about-me-list-item">Codekit</li>
-      <li class="about-me-list-item">Github</li>
-      <li class="about-me-list-item">Codepen</li>
-      <li class="about-me-list-item">Gitlab</li>
-      <li class="about-me-list-item">Terminal</li>
-    </ul>
-
+    <div class="about-me-lists-container">
+      <ul class="about-me-lists">
+        <li class="about-me-list-header">
+          <div class="star"></div>
+          <h1>Language</h1>
+        </li>
+        <li class="about-me-list-item">Javascript</li>
+        <li class="about-me-list-item">Ruby</li>
+        <li class="about-me-list-item">Html</li>
+        <li class="about-me-list-item">Css</li>
+      </ul>
+      <ul class="about-me-lists">
+        <li class="about-me-list-header">
+          <div class="square"></div>
+          <h1>Frameworks</h1>
+        </li>
+        <li class="about-me-list-item">Bootstrap</li>
+        <li class="about-me-list-item">Ruby on Rails</li>
+        <li class="about-me-list-item">RSpec</li>
+        <li class="about-me-list-item">CapyBara</li>
+        <li class="about-me-list-item">Selenium</li>
+      </ul>
+      <ul class="about-me-lists">
+        <li class="about-me-list-header">
+          <div class="circle"></div>
+          <h1>Skills</h1>
+        </li>
+        <li class="about-me-list-item">Codekit</li>
+        <li class="about-me-list-item">Github</li>
+        <li class="about-me-list-item">Codepen</li>
+        <li class="about-me-list-item">Gitlab</li>
+        <li class="about-me-list-item">Terminal</li>
+      </ul>
+    </div>
   </section>
   <section class="contact-form">
     <form class="my-form" action="https://formspree.io/f/mrgdbqpg" method="POST">
-      <p class="contact-form-content">I'm always interested in hearing about new projects, so if you'd like to chat
+      <p class="contact-form-content">
+        I'm always interested in hearing about new projects, so if you'd like to chat
         please get in touch.</p>
-      <input class="contact-form-name" type="text" name="name" placeholder="Full name" maxlength="30" required>
-      <input class="contact-form-email" type="email" name="email" placeholder="Email address" required>
-      <textarea class="contact-form-message" rows="3" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
-      <button class="contact-form-button" type="submit">Get in touch</button>
+      <div class="contact-form-inputs">
+        <input class="contact-form-name" type="text" name="name" placeholder="Full name" 
+        maxlength="30" required>
+        <input class="contact-form-email" type="email" name="email" 
+          placeholder="Email address" required>
+        <textarea class="contact-form-message" rows="3" name="message" 
+          placeholder="Enter text here" maxlength="500" required></textarea>
+        <button class="contact-form-button" type="submit">Get in touch</button>
+      </div>
     </form>
   </section>
   <footer>
```

Comment:
Please, also modify the color of the horizontal rule at the footer to match your chosen Figma template. Check my comment above for more reasons you should consider staying close to given design.#

Your design:
![image](https://user-images.githubusercontent.com/20554583/187030072-66dd9bf4-3a5f-492a-bbdb-e24c77daf29c.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/187030059-17ac5715-5402-4be5-8ab0-f32841f1a6b0.png)